src/metrics/utils.py

```diff
@@ -0,0 +1,41 @@
+import numpy as np
+import open3d as o3d
+
+from nptyping import Float, NDArray, Shape
+from src.core import Database
+
+
+def build_map(
+    db: Database,
+    min_bound: NDArray[Shape["3"], Float],
+    max_bound: NDArray[Shape["3"], Float],
+    voxel_size: float,
+) -> o3d.geometry.PointCloud:
+    global_pcd = o3d.geometry.PointCloud()
+    for pose, pcd, i in zip(db.trajectory, db.pcds, range(len(db))):
+        pcd = pcd.read().transform(pose)
+        global_pcd += pcd
+        if i % 100 == 0:
```

Comment:
I am not sure about chosen step for down sampling

Reply:
Let's make it configurable at least, no magic constants should be there